### DISCUSSION TITLE
Technology Transfer for kickers?

### DISCUSSION THREAD

**Discussion by peyre** [raceintospace/raceintospace#559] [Ideas]
I know it's very rare for a player to develop both the Kicker-A and Kicker-B, but if someone did, should they perhaps give technology transfer to one another?  Like if your A is ≥75%, a newly purchased B would start out at 40% or something?

For that matter, how about the two LMs?  If you researched the Eagle but then changed your mind and bought the Cricket, should it start at something like 25%?

Then again, this sort of change to the mechanics might require a lot of testing.

**Comment by hweimer** [comment]
There is already tech transfer for the kickers, but it's coming from rockets. Strangely enough, Titan/Proton transfers to Kicker-A, while Atlas/R-7 goes into Kicker-B (and not the other way round). What is the reason for that?

**Reply by peyre** [reply]
Whoa - I never noticed that!  And I modified that code for v. 1.1, too!  That must be an oversight or something.  It makes sense that a kicker would get its tech transfer from rocketry, but they should both get it from any of the rockets, I would think, except boosters.

It looks like the Kicker-C gets the same amount from R-7 as from the Proton, so maybe it should be an equal amount from any rocket, even the "mega" ones for direct ascent (just in case).

**Reply by smudog** [reply]
I think (going back to #519) Kicker-A is a proxy for the hypergolic Agenda (Titan was hypergolic) and Kicker-B is a proxy for Centaur (which while hydrolox compared to the kerolox Atlas, was paired with the Atlas through the sixties). The game's Titan + Kicker-B might be a good analog for the original Saturn 1 configuration.

**Reply by peyre** [reply]
Oh, that could be.  As it happens, the Proton is also hypergolic, and the R-7 cryogenic.  Not sure about the upper Blok stages they used though.

**Reply by smudog** [reply]
The Korolev design school (Vostok, Soyuz, N-1) tended toward kerolox (which was used in the Block D and Block G upper stages on the N-1), the Chelomey bureau (Proton and the rest of the Universal Rocket series) preferred hypergolic (which Korolev hated) and would later produce the hypergolic Fregat upper stage.   I believe the game approximately does Korolev for LOR, and Chelomey for direct.

**Reply by peyre** [reply]
Yes, that sounds right.  Plus Saturn was kerelox I think, and N1 as well.  The Nova would likely have been the same, but the UR-700 would've been hypergolic.

**Reply by smudog** [reply]
Saturn was first stage kerolox, and then upper stages hydrolox (much more performance but much more difficult to manage and its success in Apollo arguably doomed the next 40 years of human space flight).  N1 was all kerosene/oxygen, but small thrust chambers which introduced another dimension of complexity.  Nova would of had a substantial hydrogen component (the closest that got to a real design was would have been a Saturn-VIII). Agree on the UR-700.  In the context of this discussion, I think the 'lox' part of the technology transfers.

**Reply by hweimer** [reply]
@smudog:
> I think (going back to #519) Kicker-A is a proxy for the hypergolic Agenda (Titan was hypergolic) and Kicker-B is a proxy for Centaur (which while hydrolox compared to the kerolox Atlas, was paired with the Atlas through the sixties). The game's Titan + Kicker-B might be a good analog for the original Saturn 1 configuration.

Ah, thanks for the explanation! Wouldn't it make sense then to require a Kicker-A/B when using the lunar Atlas option with Ranger and Surveyor, respectively (although the payload masses would need to be adjusted as well)?

**Comment by sflynn850** [comment]
Is there any reason we couldn't refer to the kickers by historical names?

**Reply by smudog** [reply]
It starts to get a little fuzzy - in particular if you are trying to balance the two sides.  Is the Saturn S-IVB a kicker, for instance?  It never shows up directly in the game, but S-IVB versus Centaur was a critical early trade in reality.  There was no equivalent on the Soviet side that I was aware of, beyond the higher level Korolev/Chelomey tensions.

**Reply by peyre** [reply]
I suppose we could refer to them by function, call them something like "TLI STAGE 1" and "TLI STAGE 2".  One complication would then be what to call the Kicker-C.  I never liked the name much, but when I asked Erik Anderson about it he recommended keeping the name.  It does have a kicker/TLI stage, but also all the gear for landing the capsule.  Plans for the approach using it had the preliminary name "LK-1963" with what we call the "Kicker-C" + trans-lunar acceleration stage + maneuvering tug together called the 19K vehicle.  [https://www.russianspaceweb.com/19k.html](url)  It would have been a form of EOR, at least in the historical meaning - which is a bit different than what (BA)RIS means by EOR.

Edit: that doesn't sound right, "TLI STAGE 1" etc.  That makes it sound like you would use both on a launch together.  Maybe instead "TLI STAGE SMALL" and "TLI STAGE LARGE" or something - but that doesn't sound right either.

Edit: Or "TLI STAGE - SINGLE" and "TLI STAGE - DOUBLE" might do the trick.  Doesn't have much ring to it though.
